### ISSUE TITLE
Exception: 'Incorrect EPUB spine: item with IdRef = "cover" is missing in the manifest.'

### ISSUE THREAD

**Issue by sattasundar** [vers-one/EpubReader#109]
## Description
Getting below exception while reading epub file

`VersOne.Epub.EpubPackageException: 'Incorrect EPUB spine: item with IdRef = "cover" is missing in the manifest.' `

**Comment by daviddphillips** [comment]
I'm seeing this too:
VersOne.Epub.EpubPackageException: Incorrect EPUB manifest: item with ID = "cover" is missing.
VersOne.Epub.Utils.TaskExtensionMethods.ExecuteAndUnwrapAggregateException[T]

**Comment by vers-one** [comment]
Yeah, this is the most common problem among the EPUB books. I'll add a workaround in the next release.

**Comment by vers-one** [comment]
Implemented in version 3.3.3.

@daviddphillips, for the `Incorrect EPUB manifest: item with ID = "cover" is missing` error: these errors will be automatically ignored, if your application doesn't pass a custom `EpubReaderOptions` instance into a `EpubReader` method. Otherwise, you will need to set `EpubReaderOptions.BookCoverReaderOptions.Epub2MetadataIgnoreMissingManifestItem` to `true`.

@sattasundar, for the `Incorrect EPUB spine: item with IdRef = "cover" is missing in the manifest` error: it's not a common issue among the EPUB books, so it's not ignored by default. To ignore these errors you can use the following code:
```cs
EpubReaderOptions options = new(EpubReaderOptionsPreset.RELAXED);
options.SpineReaderOptions.IgnoreMissingManifestItems = true;
EpubBook book = EpubReader.ReadBook(filePath, options);
```
